compiler/optimizer/Inliner.cpp

```diff
@@ -6376,6 +6376,15 @@ OMR_InlinerUtil::computePrexInfo(TR_CallTarget *target)
    return NULL;
    }
 
+/**
+ * Check if targeted inlining is needed if callee is inlined
```

Comment:
What is "targeted inlining" ?  A description of that is needed in this comment if this is going to be a useful API.

Reply:
I've made the change, could you continue the review?